heartbeat/monitors/active/http/check.go

```diff
@@ -132,14 +134,100 @@ func checkHeaders(headers map[string]string) respValidator {
 	}
 }
 
-func checkBody(matcher []match.Matcher) bodyValidator {
-	return func(r *http.Response, body string) error {
-		for _, m := range matcher {
-			if m.MatchString(body) {
+func parseBody(b interface{}) ([]match.Matcher, []match.Matcher) {
+	var (
+		pm, nm []match.Matcher
+		positive = "positive"
+		negative = "negative"
+	)
+	// run through this code block if there is no positive or negative keyword in response body
+	// in this case, there's only plain body
+	p, ok := b.([]interface{})
```

Comment:
Can we combine this line and the following? That's more common in our code.

`if p, ok := b.([]interface{}); ok {` is what I'm looking for

Reply:
Goot catch, thanks for your suggestion.

Reply:
Good catch, thanks for your suggestion.